iam/snippets/service_accounts_enable.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START iam_enable_service_account]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"golang.org/x/oauth2/google"
+	iam "google.golang.org/api/iam/v1"
+)
+
+// enableServiceAccount enables a service account.
+func enableServiceAccount(w io.Writer, email string) error {
+	client, err := google.DefaultClient(context.Background(), iam.CloudPlatformScope)
+	if err != nil {
+		return fmt.Errorf("google.DefaultClient: %v", err)
+	}
+	service, err := iam.New(client)
+	if err != nil {
+		return fmt.Errorf("iam.New: %v", err)
+	}
+
+	request := new(iam.EnableServiceAccountRequest)
```

Comment:
```suggestion
	req := &iam.EnableServiceAccountRequest{}
```

Does the same thing, but follows the pattern of other samples.

Reply:
Done